UefiCpuPkg/PiSmmCpuDxeSmm/CpuS3.c

```diff
@@ -250,15 +250,3 @@ InitSmmS3ResumeState (
     InitSmmS3Cr3 ();

```

Comment:
Above this function, the following line can be remoted.  InitSmmS3Cr3() will update SmmS3ResumeState->SmmS3Cr3
SmmS3ResumeState->SmmS3Cr3 = Cr3;

Reply:
Talked with Dun, we agree clean in the furture.